src/mame/skeleton/easy_karaoke.cpp

```diff
@@ -535,6 +535,43 @@ ROM_START( karams )
 	ROM_LOAD( "ics0300-a.u9", 0x000000, 0x800000, CRC(32a7a429) SHA1(ed219bc9201b45f67c5e7dbe3fb3db70823c59f0) )
 ROM_END
 
+/*
+The 'dks7000c' set has the following 25 songs built in.
+
+A Spoonful of Sugar                         Mary Poppins
+A Whole New World                           Disney
+Bare Necessities                            The Jungle Book
+Be Our Guest                                Beauty and the Beast
+Bibbidi-Bobbidi-Boo                         Disney
+Can You Feel The Love Tonight               The Lion King
+Chim Chim Cheree                            Disney
+Circle of Life                              The Lion King
+Colors of the Wind                          Pocahontas
+Cruella De Vil                              101 Dalmations
+Hakuna Matata                               The Lion King
+I Just Can't Wait To Be King                The Lion King
+I Wan'na Be Like You (The Monkey Song)      The Jungle Book
+I Won't Say (I'm In Love)                   Hercules
+I'll Make A Man Out Of You                  Mulan
+If I Didn't Have You                        Monsters Inc.
+It's A Small World                          New York World's Fair
+Kiss The Girl                               The Little Mermaid
+Part Of Your World                          The Little Mermaid
+Supercalifragilisticexpialidocious          Disney
+Yo Ho! (A Pirate's Life For Me)             Pirates of the Caribbean
+You Can Fly! You Can Fly! You Can Fly!      Peter Pan
+You'll Be In My Heart                       Tarzan
+You've Got A Friend In Me                   Toy Story
+Zip-A-Dee-Do-Dah                            Song of the South
```

Comment:
Why do some songs just list Disney and not the film?  (A Whole New World from Aladdin Bibbidi-Bobbidi-Boo from Cinderella, etc.)